nixos/modules/services/system/dbus.nix

```diff
@@ -39,12 +39,14 @@ in
 
       implementation = mkOption {
         type = types.enum [ "dbus" "broker" ];
-        default = "broker";
+        default = "dbus";
         description = ''
           The implementation to use for the message bus defined by the D-Bus specification.
           Can be either the classic dbus daemon or dbus-broker, which aims to provide high
           performance and reliability, while keeping compatibility to the D-Bus
           reference implementation.
+          dbus-broker currently has some issues on NixOS, which is why it's
+          considered experimental and not the default for now.
```

Comment:
What issues exactly? This vague comment isn't helpful for the reader.

I've been running this since months without any issues on all my systems, so it cannot be that experimental.

Reply:
I can also entirely drop this line (it wasn't part of the revert per se). We can create a tracking issue to re-enable it past the next branchoff.

Reply:
Done.